database/README.md

```diff
@@ -0,0 +1,38 @@
+# Apollo - Database
```

Comment:
This document is a bit information heavy for what could be a few lines in the README as a note "for local development a PostgreSQL database can be started using Docker and docker-compose, with the following commands: {...}".

The list of requirements and separate shell script is overkill. This information is already present on the Docker website (and is liable to change), and the script isn't comprehensive enough to substitute for a docker-compose cheat sheet (start, stop, logs, etc.) docker-compose commands will also work in Windows CMD, while the shell script won't.

Reply:
I agree. I've updated it. Sometimes less is better